CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,113 @@
+This document was copied from https://raw.githubusercontent.com/publiclab/plots2/master/CODE_OF_CONDUCT.md on January 28, 2019. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document. 
```

Comment:
Oh! Just noted this change after the merge :disappointed:. I think we need to use original content of  https://github.com/publiclab/plots2/blob/master/CODE_OF_CONDUCT.md with change in date only. @nkleinbaer can you open a PR for it so as to change the link? Thanks!